nicegui/elements/input.py

```diff
@@ -51,6 +53,20 @@ def toggle_type(_):
 
         self.validation = validation
 
+        if autocomplete is not None:
+            def AutoCompleteInput():
+                global wordcomplete
```

Comment:
Where is wordcomplete coming from?

Reply:
Ah ok. You can have multiple PRs by creating separate branches on your forked repository.

Reply:
> Btw, what do you think about that?

About what? Overall I think the auto completion is a great feature.